scripts/src/main/resources/scripts/command/ide

```diff
@@ -303,6 +303,32 @@ function doUninstall() {
   echo "Goodbye and have a nice day!"
 }
 
+function doUpdateProjects() {
```

Comment:
you need to call this function especially from `setup` and not only from update. Also default `update` should IMHO update projects.

Reply:
Done